contrib/debian/examples/bitcoin.conf

```diff
@@ -67,9 +67,8 @@
 # This option can be specified multiple times (default: bind to all interfaces)
 #rpcbind=<addr>
 
-# You must set rpcuser and rpcpassword to secure the JSON-RPC api
-#rpcuser=Ulysseys
-#rpcpassword=YourSuperGreatPasswordNumber_DO_NOT_USE_THIS_OR_YOU_WILL_GET_ROBBED_385593
+# You must set rpcauth to secure the JSON-RPC api
+#rpcauth=<result from ./share/rpcuser/rpcuser.py User01>
```

Comment:
Anyone knows where I could find this folder? Doesn't look like it's in the ppa...